contracts/modules/social_recovery/SocialRecoveryModule.sol

```diff
@@ -294,6 +295,15 @@ contract SocialRecoveryModule is GuardianStorage {
         emit RecoveryCanceled(msg.sender, walletsNonces[msg.sender] - 1);
     }
 
+    /**
+     * @notice Invalidates the wallet's nonce to prevent replay attacks.
```

Comment:
Nit: Not a fan of the documentation. The `nonce` is there to prevent replay attacks. The invalidation function is there to increment the nonce and invalidate existing recovery confirmations.

```suggestion
     * @notice Invalidates the wallet's nonce. This will invalidate existing
     * recovery confirmations from guardians and can be used either to cancel
     * the process of collecting confirmations from guardians or when rotating
     * the guardian configuration to prevent "shadow" confirmations.
```